app/src/main/java/com/conupods/OutdoorMaps/View/MapsActivity.java

```diff
@@ -203,4 +219,33 @@ public void onRequestPermissionsResult(int requestCode, @NonNull String[] permis
             }
         }
     }
+
+    public void triggerActivityTransition(Context context){
```

Comment:
I don't think you need to pass context to this function since you can simply call it with App.getContext, app is a static class inside com.conupods. Also you are not even using the variable context in this function! FYI context is apparently a very heavy object to pass around, I think I read somewhere that it shouldn't be passed but just called by the static class instead!

Reply:
+1

Reply:
removed Context object